chart/values.yaml

```diff
@@ -3,11 +3,15 @@
 # Declare variables to be passed into your templates.
 
 mayastorCP:
-  registry:
+  registry: ''
+  tag: latest
```

Comment:
I'm not sure if latest is the correct default here, as latest is the latest release, and the chart in develop might not be compatible with that version, so maybe this should be develop?

Reply:
`latest` is the latest stable release, right? So I think the default value should be `latest` here. `develop/values.yaml` and `test/values.yaml` already override the `tag`.

Reply:
`latest` is the latest released version.
Let's say on develop we add a new cmdline argument - the `latest` will not be aware of this new argument, and install will likely fail. 
So maybe the master branch should usage tag `latest` and the develop branch should use the tag `develop`?

Reply:
> So maybe the master branch should usage tag `latest` and the develop branch should use the tag `develop`?

Yes, I guess so. Not sure what the best way is to accomplish that though.

Reply:
ugly suggestion - we could have a script that checks if the tag is "correct" - we could add it as a pre-commit hook and also to the jenkins pipeline.

Reply:
I think actually the master branch should have the exact version and `pullPolicy: IfNotPresent`. So basically what is contained in the generated files in `deploy` should already be present in `chart` as well.